components/sections/profile/settings/Linking.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+    <ProfileSettingsSection
+      title="Account Linking"
+      see-more
+      see-all
+    >
+      <div class="grid grid-cols-2 gap-4">
+        <div
+          class="flex text-gray-500 text-base font-normal"
+        >
+            <div class="pr-2 text-sm flex items-center"><DiscordIcon /></div>
+            <div class="text-sm self-center">{{ $t('profile.edit.discord') }}</div>      
+        </div>
+        <div class="flex justify-end text-primary text-sm">
```

Comment:
We should link this document with the actual discord connect status. in the profile header you can find the related attributes that can be used to show the right state